pkg/config/application_ecs.go

```diff
@@ -34,9 +38,21 @@ func (s *ECSApplicationSpec) Validate() error {
 }
 
 type ECSDeploymentInput struct {
+	// The name of network configuration file placing in application directory.
+	VpcConfigurationFile string `json:"vpcConfigurationFile"`
+	// The Amazon Resource Name (ARN) that identifies the cluster.
+	ClusterArn string `json:"clusterArn"`
+	// The launch type on which to run your task.
+	// If a launchType is specified, the capacityProviderStrategy parameter must be omitted.
+	LaunchType string `json:"launchType"`
+	// VpcConfiguration ECSVpcConfiguration `json:"awsvpcConfiguration"`
+	AwsVpcConfiguration types.AwsVpcConfiguration `json:"awsvpcConfiguration"`
+	// The capacity provider strategy to use for the task.
+	// If a capacityProviderStrategy is specified, the launchType parameter must be omitted.
+	CapacityProviderStrategy []types.CapacityProviderStrategyItem `json:"capacityProviderStrategy"`
```

Comment:
Is this duplicated with the LaunchType? Should we keep only LaunchType available to simplify the configuration?

Reply:
The difference is whether to use `CapacityProviderStrategy`

>Should we keep only LaunchType available to simplify the configuration?

I agree. I found the current configuration a bit complicated!